package.json

```diff
@@ -1,32 +1,39 @@
 {
-  "name": "bench-node",
-  "version": "0.5.0",
-  "description": "",
-  "main": "lib/index.js",
-  "scripts": {
-    "test": "node --test --allow-natives-syntax"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/RafaelGSS/bench-node.git"
-  },
-  "keywords": [
-    "benchmark",
-    "nodejs"
-  ],
-  "author": "RafaelGSS <rafael.nunu@hotmail.com>",
-  "contributors": [
-    {
-      "name": "H4ad",
-      "author": true
-    }
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/RafaelGSS/bench-node/issues"
-  },
-  "homepage": "https://github.com/RafaelGSS/bench-node#readme",
-  "dependencies": {
-    "piscina": "^4.8.0"
-  }
+	"name": "bench-node",
+	"version": "0.5.0",
+	"description": "",
+	"main": "lib/index.js",
+	"scripts": {
+		"test": "node --test --allow-natives-syntax",
+		"lint": "biome lint .",
```

Comment:
From what I remember, it's a good idea to keep separated the lint for CI and lint for local machine, so create a lint for ci and call: `biome ci .`